core/trino-main/src/main/java/io/trino/connector/ConnectorManager.java

```diff
@@ -127,7 +129,7 @@
     public ConnectorManager(
             MetadataManager metadataManager,
             CatalogManager catalogManager,
-            AccessControlManager accessControlManager,
+            EventListenerManager eventListenerManager,
```

Comment:
Why did the order of these arguments change? Maybe keep them in the same place to make the diff cleaner.